charts/everest/templates/hooks.yaml

```
@@ -1,3 +1,8 @@
 {{- if .Values.upgrade.preflightChecks }}
 {{- include "everest.preUpgradeChecks" (dict "namespace" (include "everest.namespace" .) "version" .Chart.Version "versionMetadataURL" (include "everest.versionMetadataURL" .)) }}
 {{- end }}
+---
+{{- include "everest.pspCleanup" (dict "namespace" (include "everest.namespace" .)) }}
+---
+{{- include "everest.assignDefaultPsp" (dict "namespace" (include "everest.namespace" .)) }}
```

Since we should run this only in 1.7.0 ... I know we plan to remove it in 1.8.0, but adding this condition won't harm, in case we forget to. It also acts as self-explanatory documentation 😅 
```suggestion
{{- if eq .Chart.Version "1.7.0" }}
{{- include "everest.assignDefaultPsp" (dict "namespace" (include "everest.namespace" .)) }}
{{- end }}
```

---

Done